### DISCUSSION TITLE
Is possible change the primary color without create another theme?

### DISCUSSION THREAD

**Discussion by cod3rshotout** [MahApps/MahApps.Metro#4175] [Q&A]
I'm using this theme:

    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Themes/Light.Blue.xaml" />

I would like to change the primary color which is `#2196f3` with `#00589f` that is the color of: `MahApps.Brushes.Highlight`.

How can I do this?

**Comment by cod3rshotout** [comment]
@punker76 I also tried to set `<Color x:Key="Theme.PrimaryAccentColor">#00589f </Color>` but nothing happen

**Comment by timunie** [comment]
Hi @cod3rshotout 

please read this [guide](https://github.com/ControlzEx/ControlzEx/blob/develop/Wiki/ThemeManager.md) to understand the theming in MahApps. I would suggest to generate your theme at runtime (also explained in the guide)

To get an overview of the existing `Brushes` to override please see : [/Styles/Themes/Theme.Template.xaml](https://github.com/MahApps/MahApps.Metro/blob/develop/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml)

Happy coding 
Tim

**Reply by cod3rshotout** [reply]
> Hi @cod3rshotout
> 
> please read this [guide](https://github.com/ControlzEx/ControlzEx/blob/develop/Wiki/ThemeManager.md) to understand the theming in MahApps. I would suggest to generate your theme at runtime (also explained in the guide)
> 
> To get an overview of the existing `Brushes` to override please see : [/Styles/Themes/Theme.Template.xaml](https://github.com/MahApps/MahApps.Metro/blob/develop/src/MahApps.Metro/Styles/Themes/Theme.Template.xaml)
> 
> Happy coding
> Tim

Thanks for the answer. I just need to know if I can change the primary color of the theme using an App.xaml key brush, is it possible?
